### PR TITLE
Feature/revsdl 2375 vr proxy

### PR DESCRIPTION
**Pull request by bonewell**

### CODE REVIEW

File: src/components/vr_module/include/vr_module/vr_proxy.h [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  /**
+   * Sends message to HMI(Applink)
+   * @param message to send
+   * @return true if success
+   */
+  bool Send(const vr_hmi_api::Message& message);
+
+ private:
+  void Handle(vr_hmi_api::Message message);
+  VRProxyListener *listener_;
```
Comment (DKlimenko) [inline src/components/vr_module/include/vr_module/vr_proxy.h]: is it possible to move * to type?

**Comment by DKlimenko** [comment]
reviewed

File: src/components/vr_module/include/vr_module/vr_proxy.h (hunk 1 above)
Comment (bonewell) [inline src/components/vr_module/include/vr_module/vr_proxy.h]: Possible but Google style says "When declaring a pointer variable or argument, you may place the asterisk adjacent to either the type or to the variable name". So I don't see reason to change it.

File: src/components/vr_module/include/vr_module/vr_module.h [hunk 2]
```diff
@@ -72,6 +75,9 @@ class VRModule
   static uint32_t GetNextCorrelationID() {
     return next_correlation_id_++;
   }
+
+  virtual void OnReceived(const vr_hmi_api::Message& message);
```
Comment (KKarlash) [inline src/components/vr_module/include/vr_module/vr_module.h]: could you please add description?

**Comment by KKarlash** [comment]
Reviewed

File: src/components/vr_module/include/vr_module/vr_module.h (hunk 2 above)
Comment (bonewell) [inline src/components/vr_module/include/vr_module/vr_module.h]: @KKarlash The description is updated

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**